tests/guardrails/correction_range_schedule.tests.js

```diff
@@ -0,0 +1,111 @@
+const { limits } = require('../../src/index');
+
+var correctionRangeScheduleTests = (test) => {
+    it('cannot set above max', async () => {
+        var screen = await test.settingsScreen.OpenCorrectionRangeScreen();
+        try {
+            await screen.Apply({
+                time: '12:00 AM',
+                min: limits.correctionRange.max.limit,
+                max: limits.correctionRange.max.limit + limits.correctionRange.step
+            });
+        } catch (error) {
+            await screen.Cancel();
+        }
+    });
+    it('can set at max', async () => {
+        var screen = await test.settingsScreen.OpenCorrectionRangeScreen();
+        await screen.Apply({
+            time: '12:00 AM',
+            min: limits.correctionRange.max.limit,
+            max: limits.correctionRange.max.limit,
+        });
+        //TODO assert warning
+        await screen.Cancel();
+    });
+    it('can set below max', async () => {
+        var screen = await test.settingsScreen.OpenCorrectionRangeScreen();
+        await screen.Apply({
+            time: '12:00 AM',
+            min: limits.correctionRange.max.limit,
+            max: limits.correctionRange.max.limit - limits.correctionRange.step
```

Comment:
Is this disallowed anyhow because min > max? Perhaps it would be best to test one (either min or max) at a time? Perhaps setting the non-tested one to a valid, non-warning value?